frontend-react/src/layouts/Markdown/MarkdownLayout.tsx

```diff
@@ -0,0 +1,267 @@
+                                                {b.href ? (
+                                                    <USCrumbLink href={b.href}>
+                                                        {b.label}
+                                                    </USCrumbLink>
+                                                ) : (
+                                                    b.label
+                                                )}
+                                            </Breadcrumb>
+                                        ))}
+                                    </BreadcrumbBar>
+                                ) : null}
+                                <hgroup>
+                                    <h1>{title}</h1>
+                                    {sub}
+                                </hgroup>
+                                {callToAction
+                                    ? callToAction.map((c) => (
+                                          <CallToAction key={c.label} {...c} />
+                                      ))
+                                    : null}
+                                {lastUpdated ? (
+                                    <p className="text-base text-italic">
+                                        Last updated: {lastUpdated}
+                                    </p>
+                                ) : null}
+                            </header>
+                            {tableOfContents ? (
+                                <>
+                                    <b>On this page:</b>
+                                    {tableOfContents}
+                                    <hr />
+                                </>
+                            ) : null}
+                            <MDXProvider
+                                {...mdx}
+                                components={{
+                                    ...MDXComponents,
+                                    LayoutSidenav,
+                                    LayoutMain,
+                                    ...mdx?.components,
+                                }}
+                            >
+                                {mainContent ?? children}
+                            </MDXProvider>
+                            {backToTop ? (
+                                <p>
+                                    <USSmartLink href="#top">
+                                        Back to top
+                                    </USSmartLink>
+                                </p>
+                            ) : null}
+                        </main>
+                    )}
+                </Grid>
+            </GridContainer>
+        </MarkdownLayoutContext.Provider>
+    );
+}
+
+export default MarkdownLayout;
+
+/**
+ * Creates react-router-compatible lazy function for provided file path.
+ */
+export function lazyRouteMarkdown(path: string) {
+    return async () => {
```

Comment:
Since this function `lazyRouteMarkdown` isn't used here, should it live in `AppRouter`?

Reply:
Fair point. Will rethink the location.

Reply:
Agree with moving, but into its own utils module (with the MDX imports module moved there as well) as it has an associated test module.